packages/cadl-ranch-specs/http/models/readonly-properties/main.cadl

```diff
@@ -79,4 +79,4 @@ op getOptionalPropertyModel(): OutputModel | ErrorResponse;
 @scenarioDoc("Generate, send, and receive round-trip model with readonly properties.")
 @route("/models")
 @put
-op setOptionalPropertyModel(@body input: RoundTripModel): RoundTripModel | ErrorResponse;
```

Comment:
All of `RoundTripModel`'s properties are read-only, thus not suitable for request body.